### ISSUE TITLE
"No available runtimes could be found for" issue seems to be back with ios-sim 6.1.2 and Xcode 9.1

### ISSUE THREAD

**Issue by ForestRingGames** [ios-control/ios-sim#226]
# Expected behavior

When trying to install an iOS Simulator build on the same iMac that it was built on, it should install and run. This works for older Simulators (e.g. iOS 9 iPhone 4 Simulator). Not for newer ones.

# Actual behavior.

Getting this error: No available runtimes could be found for "iPad Pro (12.9-inch) (2nd generation)".

```
Maxs-iMac:Debug-iphonesimulator Max$ ios-sim showdevicetypes
iPhone-4s, 9.2
iPhone-5, 9.2
iPhone-5s, 9.2
iPhone-5s, 11.1
iPhone-6, 9.2
iPhone-6, 11.1
iPhone-6-Plus, 9.2
iPhone-6-Plus, 11.1
iPhone-6s, 9.2
iPhone-6s, 11.1
iPhone-6s-Plus, 9.2
iPhone-6s-Plus, 11.1
iPad-2, 9.2
iPad-Retina, 9.2
iPad-Air, 9.2
iPad-Air, 11.1
iPad-Air-2, 9.2
iPad-Air-2, 11.1
iPad-Pro--9-7-inch-, 9.2
iPad-Pro--9-7-inch-, 11.1
Apple-TV-1080p, tvOS 11.1
Apple-TV-4K-4K, tvOS 11.1
Apple-TV-4K-1080p, tvOS 11.1
Apple-Watch-38mm, watchOS 4.1
Apple-Watch-42mm, watchOS 4.1
Apple-Watch-Series-2-38mm, watchOS 4.1
Apple-Watch-Series-2-42mm, watchOS 4.1
Apple-Watch-Series-3-38mm, watchOS 4.1
Apple-Watch-Series-3-42mm, watchOS 4.1
iPhone-7, 11.1
iPhone-7-Plus, 11.1
iPhone-8, 11.1
iPhone-8-Plus, 11.1
iPhone-SE, 11.1
iPhone-X, 11.1
iPad--5th-generation-, 11.1
iPad-Pro, 11.1
iPad-Pro--12-9-inch---2nd-generation-, 11.1
iPad-Pro--10-5-inch-, 11.1

Maxs-iMac:Debug-iphonesimulator Max$ ios-sim install TowerDuel-mobile.app/ --devicetypeid iPad-Pro--12-9-inch---2nd-generation-, 11.1
No available runtimes could be found for "iPad Pro (12.9-inch) (2nd generation)".
Maxs-iMac:Debug-iphonesimulator Max$ 
```

# Steps to reproduce the problem

See console print above.

# System Specs

Please run the commands below in your Terminal.app and include it in the issue. Check when done and include results below.

- [X] 1. system_profiler SPSoftwareDataType

Software:

    System Software Overview:

      System Version: macOS 10.12.6 (16G1036)
      Kernel Version: Darwin 16.7.0
      Boot Volume: Macintosh HD
      Boot Mode: Normal
      Computer Name: Maxs iMac
      User Name: Max Muthig (Max)
      Secure Virtual Memory: Enabled
      System Integrity Protection: Enabled
      Time since boot: 2:25

- [X] 2. ios-sim --version

6.1.2

- [X] 3. xcodebuild -version

Xcode 9.1
Build version 9B55

- [X] 4. xcode-select --print-path

/Applications/Xcode.app/Contents/Developer

- [X] 5. node --version

v8.1.0

**Comment by shazron** [comment]
Try quoting the devicetype and runtime:
`ios-sim install TowerDuel-mobile.app/ --devicetypeid "iPad-Pro--12-9-inch---2nd-generation-, 11.1"`

**Comment by shazron** [comment]
Might be fixed with #224 as well perhaps...

**Comment by shazron** [comment]
Just tested with 8.0.0-dev, I can't repro so it might be fixed with this version (upcoming release).
